data/PA/incentives.json

```diff
@@ -0,0 +1,1023 @@
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "pa_firstEnergy_residentialProductsRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 400
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$400 rebate for ENERGY STAR certified air-to-water heat pumps."
+    },
+    "start_date": "2021",
+    "end_date": "2026"
+  },
+  {
+    "id": "PA-19",
+    "authority_type": "other",
+    "authority": "pa-first-energy",
+    "eligible_geo_group": "pa-group-first-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "pa_firstEnergy_residentialProductsRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 500
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $500 rebate for ENERGY STAR certified air source heat pumps meeting qualifications. Rebate value depends on efficiency."
+    },
+    "start_date": "2021",
+    "end_date": "2026"
+  },
+  {
+    "id": "PA-20",
+    "authority_type": "other",
+    "authority": "pa-first-energy",
+    "eligible_geo_group": "pa-group-first-energy",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "pa_firstEnergy_residentialProductsRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 50
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$50 rebate for ENERGY STAR certified smart thermostats."
+    },
+    "start_date": "2021",
+    "end_date": "2026"
+  },
+  {
+    "id": "PA-21",
+    "authority_type": "other",
+    "authority": "pa-first-energy",
+    "eligible_geo_group": "pa-group-first-energy",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "pa_firstEnergy_residentialProductsRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 75,
+      "maximum": 75
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$75 rebate for ENERGY STAR certified smart thermostats for income-qualified residents."
+    },
+    "start_date": "2021",
+    "end_date": "2026",
+    "low_income": "pa-first-energy"
+  },
+  {
+    "id": "PA-22",
+    "authority_type": "other",
+    "authority": "pa-first-energy",
+    "eligible_geo_group": "pa-group-first-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "other",
+    "program": "pa_firstEnergy_residentialProductsRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 150
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$150 rebate for ENERGY STAR® certified packaged terminal heat pumps."
```

Comment:
I don't even know what this is? Since the item is `other` maybe we should skip it?